lib/config/fileconf.go

```diff
@@ -1430,6 +1430,9 @@ type Metrics struct {
 	// CACerts is a list of prometheus CA certificates to validate clients against.
 	// mTLS will be enabled for the service if both 'keypairs' and 'ca_certs' fields are set.
 	CACerts []string `yaml:"ca_certs,omitempty"`
+
+	// OptionalMetrics are metrics that can be toggled and are disabled by default
+	OptionalMetrics *service.OptionalMetrics `yaml:"optional_metrics,omitempty"`
```

Comment:
Do we really need to add the indirection with `OptionalMetrics`? Both `GRPCClientLatency` and `GRPCServerLatency` could be defined inline and would default to false if they weren't explicitly provided in the config file. That also seems to jive more with our existing optional configurations and would eliminate an additional level of indentation for us lazy folks.